codex-ui/dev/Playground.vue

```diff
@@ -197,16 +197,27 @@ const pages = computed(() => [
         onActivate: () => router.push('/components/editor'),
         isActive: route.path === '/components/editor',
       },
+
+      {
+        title: 'Alert',
+        onActivate: () => router.push('/components/alert'),
+        isActive: route.path === '/components/alert',
+      },
     ],
   },
 ]);
 </script>
 
 <style module>
 .playground {
+  display: flex;
+  flex-direction: column;
   background-color: var(--base--bg-primary);
   color: var(--base--text);
   min-height: 100%;
+  width: 100%;
+  height: 100%;
+  overflow-x: hidden;
```

Comment:
please, explain why these changes were added

Reply:
![Screenshot from 2024-09-24 05-44-44](https://github.com/user-attachments/assets/b2db0731-59d4-470b-8dad-971b37140607)

Without min-height of 100vh




![Screenshot from 2024-09-24 05-45-37](https://github.com/user-attachments/assets/43f16eff-ccfd-443e-a284-2838295727ab)


With min-height of 100%